src/Sets/Ball1.jl

```diff
@@ -278,11 +278,39 @@ A translated ball in the 1-norm.
 ### Algorithm
 
 We add the vector to the center of the ball.
+
+### Notes
+
+See also [`translate!(::Ball1, AbstractVector)`](@ref) for the in-place version.
```

Comment:
```suggestion
See also [`translate!(::Ball1, ::AbstractVector)`](@ref) for the in-place version.
```
Same for the other seven docstrings.